django_q/conf.py

```diff
@@ -39,10 +39,22 @@ class Conf:
     """
 
     try:
-        conf = settings.Q_CLUSTER
+        conf = settings.Q_CLUSTER.copy()
     except AttributeError:
         conf = {}
 
+    _Q_CLUSTER_NAME = os.getenv("Q_CLUSTER_NAME")
+    if _Q_CLUSTER_NAME and _Q_CLUSTER_NAME != conf.get("name") and \
+            _Q_CLUSTER_NAME != conf.get("cluster_name"):
+        conf["cluster_name"] = _Q_CLUSTER_NAME
+        alt_conf = conf.pop("ALT_CLUSTERS")
```

Comment:
This crashes when "ALT_CLUSTERS" has not been configured. Raising an error would probably be better, but it's not a big deal.

Reply:
I have pushed fixes to this problem in my branch, did I fail to include those fixes in this PR?

Reply:
Anything that gets pushed to the branch would show up in the PR too. Doesn't look like you pushed anything to the `multi-queue` branch after I reviewed it. I did update your branch, so you might have to pull the changes I made first and then push the changes you made to Github.

Reply:
Aha, that's true, please fix it. Thanks!